### PR TITLE
add new tool virheat

### PR DESCRIPTION
**Pull request by PlushZ**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [x] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by wm75** [comment]
Thanks @bgruening. Yes, Variant Analysis is also the Category we picked for the predecessor tool: https://toolshed.g2.bx.psu.edu/repository?repository_id=7aeb07b2faffd848

File: tools/virheat/virheat.xml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    <expand macro="biotools"/>
+    <expand macro="requirements"/>
+    <expand macro="version"/>
+    <command detect_errors="exit_code"><![CDATA[
+#set $vcfs_dir = 'vcfs_dir/'
+mkdir -p $vcfs_dir &&
+mkdir -p "./output_dir" &&
+#set $outfile = 'tmp_output.' + str($advanced.output_type)
+#for $input_file in $sinputs
+    #set $file_path = $vcfs_dir + $input_file.element_identifier
```
Comment (bgruening) [inline tools/virheat/virheat.xml:19]: please sanitize the element_identifier here, see other tools for examples

File: tools/virheat/virheat.xml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    <inputs>
+        <param name="sinputs" type="data_collection" format="vcf" label="Variant lists data" help="Select at least two datasets (or a dataset collection) with variant lists. Datasets are expected to represent individual samples and dataset names will be used as sample identifiers." multiple="true" collection_type="list" />
+        <conditional name="genome_data">
+            <param name="choice" type="select"
+                   label="Data for genomic annotation">
+                <option value="length">length of the genome</option>
+                <option value="gff3">gff3 file</option>
+            </param>
+            <when value="length">
+                <param argument="-l" name="genome_length" type="integer" min="0"/>
```
Comment (bgruening) [inline tools/virheat/virheat.xml:52]: label is missing

File: tools/virheat/virheat.xml [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            </when>
+            <when value="gff3">
+                <param argument="-g" name="gff3_file" type="data" format="gff3" label="gff3 file"/>
```
Comment (bgruening) [inline tools/virheat/virheat.xml:55]: What does the gff3 file do? The label could be improved or the help.

File: tools/virheat/virheat.xml [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+                <option value="no">No</option>
+            </param>
+            <when value="yes">
+                <param name="zoom_start" type="integer" label="Start"/> 
+                <param name="zoom_stop" type="integer" label="End"/>
+            </when>
+            <when value="no"/>
+        </conditional>
+        <param argument="--sort" type="boolean" truevalue="--sort" falsevalue="--no-sort" checked="false" label="Sort sample names alphanumerically"/>
+        <param argument="--min-cov" type="integer" value="20" label="Display mutations covered at least x time"/>
```
Comment (bgruening) [inline tools/virheat/virheat.xml:73]: min/max?

**Review by bgruening — COMMENTED** [review]
You could add a `<creator>` tag as well.

**Comment by PlushZ** [comment]
> You could add a `<creator>` tag as well.

what does it mean?

File: tools/virheat/virheat.xml (hunk 4 above)
Comment (PlushZ) [reply tools/virheat/virheat.xml]: fixed

File: tools/virheat/virheat.xml (hunk 3 above)
Comment (PlushZ) [reply tools/virheat/virheat.xml]: fixed

File: tools/virheat/virheat.xml (hunk 2 above)
Comment (PlushZ) [reply tools/virheat/virheat.xml]: fixed

File: tools/virheat/virheat.xml (hunk 1 above)
Comment (PlushZ) [reply tools/virheat/virheat.xml]: fixed

**Comment by bgruening** [comment]
https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator

**Review by bgruening — APPROVED** [review]

**Pull request merged**